src/screens/OrgList/OrgList.tsx

```diff
@@ -79,6 +79,12 @@ function OrgList(): JSX.Element {
             </a>
           </Row>
         </Navbar.Brand>
+        <button
+          className={styles.logoutbtn}
+          onClick={() => window.location.replace('/')}
```

Comment:
clear the local storage before logout

Reply:
I thought of doing it here, but then I saw the login component which clears the localstorage.

https://github.com/PalisadoesFoundation/talawa-admin/blob/35098b347817718df86e3939658c40cd17a289e0/src/screens/LoginPage/LoginPage.tsx#L13

Reply:
Can you please elaborate on the problem with reloading and localstorage?

Reply:
Considering the above issue, should i also make changes to this code here?
https://github.com/PalisadoesFoundation/talawa-admin/blob/35098b347817718df86e3939658c40cd17a289e0/src/components/AdminNavbar/AdminNavbar.tsx#L95-L98